docs/usage/assets/background-tasks.md

```diff
@@ -0,0 +1,26 @@
+# Background tasks
+Three types of tasks are done in the background:
+- rule tasks,
+- knowledge tasks,
+- user tasks.
+
+Knowledge and user tasks can be seen and manage from Data > Background Tasks.
```

Comment:
I think you should do the same as for rule tasks. For each type : say where it can be found.

Reply:
fixed